apps/prairielearn/src/pages/instructorAssessmentQuestions/instructorAssessmentQuestions.html.ts

```diff
@@ -0,0 +1,382 @@
+      }
+      if (assessmentType === 'Homework') {
+        return `${init_points - max_manual_points}/${max_auto_points}`;
+      }
+    } else {
+      return html`&mdash;`;
+    }
+  }
+
+  return html`
+    <div class="table-responsive">
+      <table class="table table-sm table-hover">
+        <thead>
+          <tr>
+            <th><span class="sr-only">Name</span></th>
+            <th>QID</th>
+            <th>Topic</th>
+            <th>Tags</th>
+            <th>Auto Points</th>
+            <th>Manual Points</th>
+            ${showAdvanceScorePercCol ? html`<th>Advance Score</th>` : ''}
+            <th width="100">Mean score</th>
+            <th>Num. Submissions Histogram</th>
+            <th>Other Assessments</th>
+            <th class="text-right">Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          ${questions.map((question, iRow) => {
+            return html`
+              ${question.start_new_zone
+                ? html`
+                    <tr>
+                      <th colspan="${nTableCols}">
+                        Zone ${question.zone_number}. ${question.zone_title}
+                        ${question.zone_number_choose == null
+                          ? '(Choose all questions)'
+                          : question.zone_number_choose === 1
+                            ? '(Choose 1 question)'
+                            : `(Choose ${question.zone_number_choose} questions)`}
+                        ${question.zone_has_max_points
+                          ? `(maximum ${question.zone_max_points} points)`
+                          : ''}
+                        ${question.zone_has_best_questions
+                          ? `(best ${question.zone_best_questions} questions)`
+                          : ''}
+                      </th>
+                    </tr>
+                  `
+                : ''}
+              ${question.start_new_alternative_group && question.alternative_group_size > 1
+                ? html`
+                    <tr>
+                      <td colspan="${nTableCols}">
+                        ${question.alternative_group_number}.
+                        ${question.alternative_group_number_choose == null
+                          ? 'Choose all questions from:'
+                          : question.alternative_group_number_choose === 1
+                            ? 'Choose 1 question from:'
+                            : `Choose ${question.alternative_group_number_choose} questions from:`}
+                      </td>
+                    </tr>
+                  `
+                : ''}
+              <tr>
+                <td>
+                  <a href="${urlPrefix}/question/${question.question_id}/">
+                    ${question.alternative_group_size === 1
+                      ? `${question.alternative_group_number}.`
+                      : `&nbsp;&nbsp;&nbsp;&nbsp; ${question.alternative_group_number}.${question.number_in_alternative_group}.`}
```

Comment:
I know this comes from the original, but it's really bad practice. Maybe add a margin or padding instead?

```suggestion
                      : html`<span class="ml-2">${question.alternative_group_number}.${question.number_in_alternative_group}.</span>`}
```

Reply:
Great suggestion! Committed in [45609c9](https://github.com/PrairieLearn/PrairieLearn/pull/9680/commits/45609c97bd15586639d69be836846627f6cc8e3f). This seems to indent correctly, as seen below: 
![Screenshot 2024-04-10 at 2 04 33 PM](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/cc1e41be-75b1-4299-9e87-8b3198185a58)

Reply:
Looks good, but maybe indent even slightly more? Maybe `ml-3`?

Reply:
@mwest1066 Expanded the margin to `ml-3` in [a120633](https://github.com/PrairieLearn/PrairieLearn/pull/9680/commits/a120633a3ee1740ebad85e69450683c44a0ac1ee). Does this look better? 
![Screenshot 2024-04-11 at 11 45 13 AM](https://github.com/PrairieLearn/PrairieLearn/assets/114321556/783d987c-62a4-4d7b-830c-154c802d2f56)

Reply:
That looks good to me!

Reply:
Also to me! Thanks!